### PR TITLE
Add QueryAssert#doesNotContain

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java
Comment (kokosing) [inline tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java:285]: `Unexpectedly` does not sound to me like assertion error.

`Unexpected rows found:`?

File: tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java
Comment (kokosing) [inline tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java:289]: it would be nice to use `appendRows` here

File: tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java
Comment (kokosing) [inline tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java:279]: `rows.stream()...colect(toImmutableList())`?

File: tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java
Comment (kokosing) [inline tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java:291]: this could be huge message

**Review by kokosing — APPROVED** [review]

**Comment by findepi** [comment]
I think it may turn out I don't need this. Since this API is slightly leaning towards error-proneness, let's not merge this now and revisit when needed.

**Pull request closed without merging**
